meshtal.py

```diff
@@ -1346,3 +1381,81 @@ def SEAM(*meshtalarray):
                     result.value[i, j, k, e] = np.dot(w, errors)/sum(w)
                     result.error[i, j, k, e] = minerror
     return result
+
+def ratio(RefTal,DivTal):
+    """ Makes a ratio between meshtally1 and meshtally2 (mesh1/mesh2).  Geometry must match or it will fail."""
+    Part = RefTal.part
+    ee = RefTal.ebins
+    print("checking tally consistency")
+
+    if not Geoeq(DivTal, RefTal):
+        print("Tally {ntally} Geometry does not match tally {nref}, adding them is beyond my current capabilites. Sorry".format(ntally=DivTal.n,nref=RefTal.n))
+        return None
+    if DivTal.part != Part:
+        confirm = input("Tally {ntally} and tally {nref}, are for different particles. Do you REALLY know what you are doing??(Y/N).".format(ntally=DivTal.n,nref=RefTal.n))
+        if confirm not in ["Y","y"]:
+            print("Cancelling")
+            return None
+        else:
+            print("Very well")
+    if not np.array_equal(DivTal.ebins, ee):
+        print("WARNING: Energy bins not matching. This can be OK, but make sure you know what you are doing!")
+    Result = copy (RefTal, exclude=['probID', 'comment', 'value', 'error'])
+    Result.probID = "Generated by meshtal class from {probID}".format(probID=RefTal.probID)
+    Result.comment = "Tallies compared: {0}/{1}".format(RefTal.n,DivTal.n)
+    for (i, j, k, e) in np.ndindex(Result.iints, Result.jints, Result.kints, Result.eints):
+        if DivTal.value[i,j,k,e] != 0:
+            Result.value[i,j,k,e] = RefTal.value[i,j,k,e]/DivTal.value[i,j,k,e]
+            if RefTal.error[i,j,k,e] == 1 or DivTal.error[i,j,k,e] == 1:
+                Result.error[i,j,k,e] = 1
+            else:
+                Result.error[i,j,k,e] = (sqrt(pow(RefTal.error[i,j,k,e],2) + pow(DivTal.error[i,j,k,e],2)))
+        else:
+            Result.value[i,j,k,e] = 0
+            Result.error[i,j,k,e] = 0
+    return Result
+
+def combine(*tallies):
```

Comment:
Mmph, no. This is basically a special case of merging. We can add this kind of functionality to SEAM() or merge(), but I really don't want this as a separate function. OR, if you want it as a separate function, have it call merge(), and change the 0's from there.